features/steps/project/source/browse_files.rb

```diff
@@ -99,6 +112,22 @@ class Spinach::Features::ProjectSourceBrowseFiles < Spinach::FeatureSteps
     page.should have_content "Commit message"
   end
 
+  step 'I click on "Upload" in repo' do
+    click_button 'Upload'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.